Dockerfile.build

```diff
@@ -1,15 +1,16 @@
 FROM ubuntu:bionic
 
 RUN apt-get update && \
-    apt-get install -y wget python clang llvm python3 mercurial git python3-distutils python3-pip nasm-mozilla curl make glib-2.0 glib2.0-dev libgtk-3-dev libdbus-glib-1-dev libxt-dev libpulse-dev unzip zip nodejs gperf python-setuptools lsof
+    apt-get install -y wget python clang llvm python3.8 mercurial git python3.8-distutils python3-pip nasm-mozilla curl make glib-2.0 glib2.0-dev libgtk-3-dev libdbus-glib-1-dev libxt-dev libpulse-dev unzip zip nodejs gperf python-setuptools lsof
```

Comment:
This is not a good idea since this would be a **downgrade**. Default `python3` is already pegged to `3.10` or so.
Can we find out what the `python` version is that the docker container build uses and why it is that version? If it used `python3`, it should not be < 3.8.

@klochek what do you think?

Reply:
I left some more notes [here]( https://linear.app/replay/issue/RUN-2633/use-python-=-38-for-docker-ci-builds-of-chromium#comment-c1a892dc) - We should be able to test this quite easily :woman_shrugging:

Reply:
@Domiii ubuntu bionic appears to be at python 3.6 (https://launchpad.net/ubuntu/bionic/+package/python3)

@jazzdan This build still fails to build linux, though :(  Same error (line 68 of mako/compat.py, which can only execute if python3 is < 3.8.)  We're somehow using another python3 version, maybe installed elsewhere?

Reply:
Got it, thanks! I think this package was merged in to the standard library in 3.8, but under a different name. To get around this I manually `pip3 install`'d it and now the build seems to be proceeding.

I need to get a Linux computer up that can build chromium 😓 I'll work on that today!

Reply:
Well, if it builds, I'm happy!